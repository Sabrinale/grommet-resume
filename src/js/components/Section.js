import React from 'react'

import { Box } from 'grommet'

import { SectionTitle } from './SectionTitle'
import { SectionContent } from './SectionContent'

export const Section = ({ title, content }) => (
  <Box
    flex
    basis="auto"
    pad={{ left: '0px', right: '0px', top: '0px', bottom: '0px' }}
    margin={{ left: '0px', right: '0px', top: 'small', bottom: 'medium' }}
  >
    <Box
      fill={true}
      align="baseline"
      overflow="hidden"
      elevation="none"
      round="none"
      gap="small"
      pad={{ left: 'xlarge', bottom: 'small' }}
      border={{}}
      margin={{}}
    >
      <Box fill={true} flex={true}>
        <Box margin={{ left: '35%' }}>
          <SectionTitle>{title}</SectionTitle>
          <SectionContent>{content}</SectionContent>
        </Box>
      </Box>
    </Box>
  </Box>
)
