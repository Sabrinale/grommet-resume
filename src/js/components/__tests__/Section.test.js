import React from 'react'
import renderer from 'react-test-renderer'

import { Section } from '../Section'

test('renders section element', () => {
  const component = renderer.create(<Section>test</Section>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
