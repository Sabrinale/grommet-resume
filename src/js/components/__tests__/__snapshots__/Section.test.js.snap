// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders section element 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 bzUwtO"
>
  <div
    className="StyledBox-sc-13pk1d4-0 fTRPed"
  >
    <div
      className="StyledBox-sc-13pk1d4-0 OaRuD"
    >
      <div
        className="StyledBox-sc-13pk1d4-0 jJCmt"
      >
        <h3
          className="StyledHeading-sc-1rdh4aw-0 jVEgcZ"
        />
        <div
          className="StyledBox-sc-13pk1d4-0 jfnaIv"
        >
          <div
            className="StyledBox-sc-13pk1d4-0 dViohz"
          >
            <div
              className="StyledBox-sc-13pk1d4-0 OaRuD"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
